### ISSUE TITLE
Ineffective block_layout settings

### ISSUE THREAD

**Issue by briggySmalls** [20Tauri/DoxyDoxygen#82]
Hi. I love this plugin! Thanks 👏

I am having some trouble configuring my own `block_layout` options. I get unexpected behaviour when adding some of the commented-out suggestions from **Doxy.sublime-settings -- Default** into **Doxy.sublime-settings -- User**.

```javascript
{
    "block_layout": {
        "Doxygen": [
            // <sample inserted here>
        ],
    }
}
```

## File Header ✘

```javascript
{
    "tags": [
        "@brief            File header",
        "",
    ],
    "context": [
        { "key": "row",      "operator": "equal",          "operand": "0" },
        { "key": "kind",     "operator": "equal",          "operand": "" }
    ]
},
```

If I type `/**` **+ tab** in the first line of a C file I am presented with the default:
```c
/**
 * { item_description }
 */
```
## Private item ✘

```javascript
{
    "tags": [
        "@brief            I'm a private item"
    ],
    "context": [
        { "key": "name",      "operator": "regex_match",    "operand": "^_.*$" },   // re.match(operand, key)
    ]
},
```

On a variable I get the standard `@brief` comment (as if the rule doesn't detect). On a function prefixed with '_' I get:

```c
/**
 * @brief      { function_description } */
```

Note the comment ending on the same line as the `@brief`. It seems the rule is kicking in but failing to insert the specified tags?

## Compact style ✔

```javascript
{
    "tags": [
        "@brief",
        "@param",
        "@tparam",
        "@return",
        ""
    ],
    "context": [
        { "key": "nb_params", "operator": "lower_than",     "operand": "2" }
    ]
}
```

This works as expected

## Other Info

All usage occurring in '.c' file with syntax set to C.

Sublime Text 3: Build 3126
DoxyDoxygen: v0.59.0

**Comment by 20Tauri** [comment]
No, your settings is look correct... and suggested configuration too.
It's simply a regression and a leak in my test suite
If there's no definition after comment, settings are simply ignored.

I will fix it in a next release

**Comment by 20Tauri** [comment]
Fix applied on 0.59.2 (evolution branch)

**Comment by briggySmalls** [comment]
I tried this out and it works! Thanks very much 🙇
